config/client_config/me/system/config/templates/features/aca_individual_market/financial_assistance.yml

```diff
@@ -138,10 +138,8 @@ registry:
     - key: :transfer_service
       is_enabled: <%= ENV['TRANSFER_SERVICE_IS_ENABLED'] || true %>
       item: "aces"
-    - key: :non_magi_medicaid_eligible
-      is_enabled: <%= ENV['NON_MAGI_MEDICAID_ELIGIBLE_IS_ENABLED'] || false %>
-    - key: :eligible_for_non_magi_reasons
-      is_enabled: <%= ENV['ELIGIBLE_FOR_NON_MAGI_REASONS_IS_ENABLED'] || false %>
+    - key: :non_magi_transfer
+      is_enabled: <%= ENV['NON_MAGI_TRANSFER_IS_ENABLED'] || false %>
```

Comment:
We should use a default value of 'true' for ME config of non_magi_transfer feature in order to preserve the current behavior for that client.